streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamTask.java

```diff
@@ -317,7 +327,8 @@ public void initTopology() {
         for (ProcessorNode node : this.topology.processors()) {
             processorContext.setCurrentNode(node);
             try {
-                node.init(this.processorContext);
+                node.init(processorContext);
+                node.nodeMetrics.nodeTaskCreationSensor.record();
```

Comment:
why not record this inside `init`? It seems wrong to expose a field of `ProcessorNode`